### PR TITLE
HTML validation

### PR DESCRIPTION
**Pull request by DominicSimpson**
Latest version of HTML that validates nearly correctly on W3C schools.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -70,21 +70,13 @@
     justify-content: space-evenly;
   }
 
-  .justify-space-around {
-    justify-content: space-around;
```
Comment (d0g3bl2) [inline css/style.css:74]: keep

File: css/style.css [hunk 2]
```diff
@@ -70,21 +70,13 @@
...
-  }
-
```
Comment (GalKJ) [inline css/style.css:76]: keep space around

File: css/style.css [hunk 3]
```diff
@@ -70,21 +70,13 @@
...
     max-width: 20rem;
   }
 
   .width-md {
     max-width: 40rem;
   }
-
-  .min-width-sm {
-    min-width: 30rem;
-  }
```
Comment (GalKJ) [inline css/style.css:87]: delete

**Review by GalKJ — APPROVED** [review]
Files viewed and edited

**Pull request merged**